src/Guard.php

```diff
@@ -107,4 +104,24 @@ protected function hasValidProvider($tokenable)
 
         return $tokenable instanceof $model;
     }
+
+    /**
+     * Determine if the provided access token is valid.
+     *
+     * @param mixed $accessToken
+     * @return bool
+     */
+    protected function isValidAccessToken($accessToken): bool
+    {
+        $is_valid = $accessToken && (
+                ($this->expiration && $accessToken->created_at->gt(now()->subMinutes($this->expiration)))
+                || $this->hasValidProvider($accessToken->tokenable)
+            );
```

Comment:
This check doesn't read the same to me. The access token must be not null and then you specify it MUST also have an explicit expiration date. But, I don't think we required tokens to have an expiration date before this.

Reply:
You're right. I tried to inverse the statement to prevent a double negation back to `valid`. But in the process I did end up changing the behavior. Was more intent on keeping the tests working. It's fixed now. Hope it makes more sense this way.

Also added an early return on a missing `$accessToken`. Because without it we can't deduce a `User` so it makes sense to short-circuit there.